src/version.js

```diff
@@ -4,3 +4,4 @@ export const VERSION = React.version;
 export const REACT013 = VERSION.slice(0, 4) === '0.13';
 export const REACT014 = VERSION.slice(0, 4) === '0.14';
 export const REACT15 = VERSION.slice(0, 3) === '15.';
+export const REACT155 = VERSION.slice(0, 4) === '15.5';
```

Comment:
well wouldn't your suggestion match `16.5.0` which is more likely than `15.50`? ;)